io.github.loot.loot.yml

```diff
@@ -0,0 +1,51 @@
+app-id: io.github.loot.loot
+runtime: org.kde.Platform
+runtime-version: '6.5'
+sdk: org.kde.Sdk
+command: LOOT
+modules:
+  - name: application
+    buildsystem: simple
+    build-commands:
+      - install -D bin/LOOT /app/bin/LOOT
+      - install -D lib/libloot.so /app/lib/libloot.so
+      - cp -r share /app
+    sources:
+      - type: archive
+        url: https://github.com/loot/loot/releases/download/0.21.0/loot_0.21.0-Linux.tar.xz
+        sha256: 05e6a4eb1cbb237f507dc39957fe6ac485b7f5f402d351caa1a5fe7dc179c869
+  - name: libtbb
+    buildsystem: simple
+    build-commands:
+      - install -D lib/intel64/gcc4.8/libtbb.so.2 /app/lib/
```

Comment:
really you should build tbb from source.

Reply:
Why?

Reply:
because it's better.

Reply:
In what way?

This approach integrates nicely with my existing CI pipeline, and I've been producing Flatpak packages as single-file bundles as part of CI and testing them over the past month or so, so I know that this libtbb binary works as expected in a Flatpak package. Building from source just seems like it would increase the complexity and running time of Flathub builds and my own CI pipelines for no obvious benefit.